crates/uv-cli/src/lib.rs

```diff
@@ -2395,6 +2395,10 @@ pub struct RunArgs {
     #[arg(long, overrides_with("dev"))]
     pub no_dev: bool,
 
+    /// Install the project and any local dependencies as non-editable.
```

Comment:
It isn't clear here what a "local" dependency is, can we just say something like "Convert any editable dependencies, including the project and any workspace members, to non-editable dependencies"?